pcsd/pcsd.8.in

```diff
@@ -45,6 +45,9 @@ List of hostnames for which proxy is not used.
 .TP
 .B PCSD_DEBUG=<boolean>
 Set to \fBtrue\fR for advanced pcsd debugging information.
+.TP
+.B PCSD_RESTART_AFTER_REQUESTS=<integer>
+Number of requests after which the Ruby server will be restarted to prevent memory leaks. To disable restarts, set 0. Values lower than 50 are interpreted as 50. Invalid values are ignored and default (200) is set instead.
```

Comment:
I would prefer "reduce memory footprint" or "limit the amount of memory consumed by (accidental) memory leak" instead of "prevent memory leaks".